src/fhirConfig.ts

```diff
@@ -10,9 +10,13 @@ import { History } from './history';
 import { Persistence } from './persistence';
 import { Search } from './search';
 
-export interface Strategy {
+export interface OAuthStrategy {
     oauthAuthorizationUrl?: string;
     oauthTokenUrl?: string;

```

Comment:
I would actually leave these to be required (remove the `?`) since the object is optional

Unless there is a use-case that would allow for the object to be there but not these params